internal/server/http/handler.go

```diff
@@ -5,11 +5,13 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"time"
 )
 
 // ImposterHandler create specific handler for the received imposter
 func ImposterHandler(imposter Imposter) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(imposter.Response.Delay.Delay())
```

Comment:
I'd try to respect the Law of Demeter (LoD), avoiding multiple "dot" calls (A.B.C.D), because it makes the code highly coupled and it can be problematic on future changes.

Reply:
The most obvious solution, that I can imagine is to add method Delay, to Response struct.